cinema/serializers.py

```diff
@@ -31,9 +31,25 @@ class Meta:
 
 
 class MovieSerializer(serializers.ModelSerializer):
+    image = serializers.ImageField(read_only=True)
+
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+            "image"
```

Comment:
As per task requirements:

> Image field should not be available on POST api/cinema/movies/.
> Image url should be shown on:
> Movie: list and detail pages

With your implementation it is being shown on every endpoint. Make sure you show it only on List/Detail

Reply:
I suggest you just remove image field from here, then add it on MovieListSerializer and MovieDetailSerializer below and so you will not need to make a new MovieCreateSerializer